storj/http.py

```diff
@@ -66,7 +66,7 @@ def authenticate(self, ecdsa_private_key=None):
         if isinstance(ecdsa_private_key, SigningKey):
             self.private_key = ecdsa_private_key
             self.public_key = self.private_key.get_verifying_key()
-            self.public_key_hex = ecdsa_to_hex(self.public_key)
+            self.public_key_hex = ecdsa_to_hex(self.public_key.to_string())
```

Comment:
`to_string()` does not exist.
it was `str(self.public_key)`.
fixing on #89 PR.